### DISCUSSION TITLE
Question about code editor

### DISCUSSION THREAD

**Discussion by a-sennov** [stride3d/stride#1336] [Q&A] [answered]
Hi, 
Quite new to Stride so maybe a noob question. I've managed to compile it with .NET 6 and almost everything is fine except one thing: code completion in integrated .cs editor (see image). Is there a way to get rid of this "predefined type not defined' nonsense? (projects compile and run without errors)

![strd](https://user-images.githubusercontent.com/3610734/154977501-2b49ce9b-1579-42f8-8d5a-3e7d941ca162.png)

Also, are there some docs about bootstrap process? From the first sight it seems like application that is built  to be portable (.NET runtime copied and used locally) but at the same time it always tries to download Stride assemblies from NuGet even when freshly built versions are present in the same directory.

**Comment by NicusorN5** [comment]
Have you tried rebuilding the solution? Cleaning the build? Cleaning the .vs folder?

**Reply by a-sennov** [reply]
All of that plus changing target .NET FW version to 4.7.2 - behavior is consistent :)

**Comment by bgsulz** [comment]
I have encountered the same issue and the same lack of success to resolve it. Do you have any update? Were you ever able to fix this issue?

**Comment by ericwj** [comment, marked as answer]
#1453
